joinDate.json

```diff
@@ -0,0 +1,8 @@
+[
```

Comment:
Refrain from committing idiosyncratic files to git. Add files to the `.gitignore` as appropriate. Also please move this to a more appropriate location, not the root folder.